### ISSUE TITLE
Thanks for posting these documents

### ISSUE THREAD

**Issue by rrobinett** [mgx0/ZMRN0808-V5#1]
I could not find them anywhere else

**Comment by mgx0** [comment]
Yeah glad that it helped you too. It's almost impossible to get it from the seller. I got it from vendor after many tries

**Comment by clements63** [comment]
Hello
I have relay card in 5.7 without issue and the status is updated in domoticz. 
in 5.9 status is not sent to domoticz 
How to proceed to flash the card in 5.7 version please ?

**Comment by mgx0** [comment]
Hi, I'm sorry, I really don't know. My board is laying in a box unused.
